src/assets/scss/_navbar.scss

```diff
@@ -268,6 +269,10 @@ nav.navbar {
         // overflow-y: auto;
         width: max-content;
         max-width: 450px;
+        
+        // setting max height for submenus
+        max-height: 50vh;
```

Comment:
This is not acceptable and has to be dynamic. Make sure you're replicating the existing behavior which is setting the height based on the available space. The scrollbar should only appear when there's a need for it.

This solution is not dynamic and doesn't work for very long submenus.

If you're using mac, go to "System preferences > General" and then make sure "Show scroll bars" is set to "Always". That way you can see that your solution is always showing the placeholder for scrollbar, which is not desirable.

Reply:
Fixed the issue